charts/backstage/README.md

```diff
@@ -86,6 +86,7 @@ The command removes all the Kubernetes components associated with the chart and
 | Key | Description | Type | Default |
 |-----|-------------|------|---------|
 | backstage | Backstage parameters | object | See below |
+| backstage.annotations | Additional custom annotations for Deployment resource | object | `{}` |
```

Comment:
```suggestion
| backstage.annotations | Additional custom annotations for the `Deployment` resource | object | `{}` |
```

Reply:
Done. Thank you! Had to be updated in `values.yaml` so `helm-docs` can pick it up.